src/p5.dance.js

```diff
@@ -685,13 +684,18 @@ module.exports = class DanceParty {
     this.setMetadata_(id, await response.json());
   }
 
-  loadDevelopmentSongs_(callback) {
-    let ids = ['macklemore90', 'hammer', 'peas'];
+  loadSongs_(callback) {
+    fetch(`/api/v1/sound-library/hoc_song_meta/songManifest.json`)
```

Comment:
Is the eventual plan to hit a different URL in test levels vs. the shipped tutorial?  Wonder if this path should be configurable (or better yet, loaded before `new DanceLab()` is called, and the manifest data passed in to the constructor instead of the manifest path).

Reply:
The eventual plan was to hit different urls depending on test vs. shipped levels. This currently unblocks the work of adding/testing new songs in the manifest.